CONTRIBUTING.adoc

```diff
@@ -5,225 +5,160 @@ OpenShift Developers <dev@lists.openshift.redhat.com>
-        # load templates
-        $ oc create -f /data/src/github.com/openshift/origin/examples/sample-app/application-template-stibuild.json -n openshift
-        $ oc create -f /data/src/github.com/openshift/origin/examples/db-templates -n openshift
-
-
-11. At this point you can open a browser on your host system and navigate to https://localhost:8443/console to view the web console.  You can log in with any username and password combination.
-
-
-12.  NOTE: to properly stop OpenShift and clean up, so that you can start fresh instance of OpenShift, execute:
-
-    # shut down openshift
-    $ sudo pkill openshift
-    # stop the docker containers
-    $ docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
-    # deleting all the internal config files, etcd, etc and starting openshift fresh
-    $ sudo rm -rf openshift.local.*
-    # if you used the --volume-dir=/home/vagrant/volumes flag, then run these
-
-TIP: See https://github.com/openshift/vagrant-openshift for more advanced options
-
-==== Ensure virtual box interfaces are not managed by Network Manager
-
-If you are developing on a Linux host, then you need to ensure that Network Manager is ignoring the
-virtual box interfaces, otherwise they cause issues with multi-vm networking.
-
-Follow these steps to ensure that virtual box interfaces are unmanaged:
-
-1. Check the status of Network Manager devices:
-
-   $ nmcli d
-
-2. If any devices whose name start with vboxnet* are not unmanaged, then they need to be added to
-   NetworkManager configuration to be ignored.
-
-   $ cat /etc/NetworkManager/NetworkManager.conf
-
-        [keyfile]
-        unmanaged-devices=mac:0a:00:27:00:00:00;mac:0a:00:27:00:00:01;mac:0a:00:27:00:00:02
-
-3. One can use the following command to help generate the configuration:
+==== Building Origin Binaries, RPMs and Images
 
-   $ ip link list | grep vboxnet  -A 1 | grep link/ether | awk '{print "mac:" $2}' |  paste -sd ";" -
+In order to build a full release of Origin, containing binaries, RPMs and
+container images, run:+
+----
+$ hack/env make release
+----
+In order to make use of the binaries from your shell, add the build output
+directory to the `$PATH`:+
+----
+$ export PATH="${PATH}:$( source hack/lib/init.sh; echo "${OS_OUTPUT_BINPATH}/$( os::util::host_platform )/" )"
+----
 
-4. Reload the Network Manager configuration:
+See more information in [`HACKING.md`](./HACKING.md#building-on-non-linux-systems)
+for a more in-depth approach to building releases and incremental artifacts.
 
-    $ sudo nmcli con reload
+==== Running the OpenShift Server
+Next, follow the set-up steps in ./docs/cluster_up_down.md[`cluster_up_down.md`]
+to start a cluster with `oc cluster up`. When starting the cluster, you will
+need to use the container images you built locally, which are tagged with
+the `git` commit you're working off of as well as `:latest`. If you have not
```

Comment:
this is ambiguous, it could be interpretted as "you need both the images you built locally(tagged with the git commit) and you need the ":latest" images"  but what you mean is "you need the images you built locally.  they are tagged as :latest and with the git commit"